gcloud/error_reporting/client.py

```diff
@@ -0,0 +1,59 @@
+#!/usr/bin/env python
+# Copyright 2016 Google Inc. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""Client for interacting with the Stackdriver Logging API"""
+
+import traceback
+
+import gcloud.logging.client
+
+DEFAULT_SERVICE = 'python'
+
+
+class Client(gcloud.logging.client.Client):
+    """Error Reporting client. Currently Error Reporting is done via the
+    Logging Client so it just subclasses that."""
+
+    def report_error(self, message="", service=DEFAULT_SERVICE):
```

Comment:
Seems like `service` would be a good property / constructor parameter for the `Client` instance, on the assumption that the part of the app where the client was constructed would correspond to a "service".  `report_error` could still take the parameter, defaulting to `None`, and use the instance property as the fallback.